Index: src/routes/authRoutes.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/routes/authRoutes.ts b/src/routes/authRoutes.ts
--- a/src/routes/authRoutes.ts	
+++ b/src/routes/authRoutes.ts	(date 1733086723986)
@@ -48,6 +48,7 @@
         await newUser.save();
 
         const token = jwt.sign({ userId: newUser._id }, process.env.JWT_SECRET || 'defaultSecret', { expiresIn: '24h' });
+        console.log('JWT_SECRET مورد استفاده در ساخت توکن:', process.env.JWT_SECRET || 'defaultSecret');
 
 
         res.status(201).json({
@@ -80,6 +81,7 @@
             }
 
         const token = jwt.sign({ userId: user._id }, process.env.JWT_SECRET || 'defaultSecret', { expiresIn: '24h' });
+        console.log('JWT_SECRET مورد استفاده در ساخت توکن:', process.env.JWT_SECRET || 'defaultSecret');
 
         res.status(201).json({
 
